quodlibet/quodlibet/browsers/covergrid/prefs.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+# Copyright 2004-2007 Joe Wreschnig, Michael Urman, Iñigo Serna
+#           2009-2010 Steven Robertson
+#      2012,2013,2016 Nick Boultbee
+#           2009-2013 Christoph Reiter
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 as
+# published by the Free Software Foundation
+
+from gi.repository import Gtk
+
+from quodlibet import config
+from quodlibet import qltk
+from quodlibet import util
+from quodlibet import _
+from quodlibet.browsers._base import FakeDisplayItem, EditDisplayPatternMixin
+from quodlibet.formats import PEOPLE
+from quodlibet.qltk import Button, Icons
+from quodlibet.qltk.ccb import ConfigCheckButton
+from quodlibet.util import format_rating
+from quodlibet.util.i18n import numeric_phrase
+
+PEOPLE
+_SOME_PEOPLE = "\n".join([util.tag("artist"), util.tag("performer"),
+                         util.tag("composer"), util.tag("arranger"), ])
+
+_EMPTY = _("Songs not in an album")
+DEFAULT_PATTERN_TEXT = """[b]<album|<album>|%s>[/b]<date| (<date>)>
+[small]<~discs|<~discs> - ><~tracks> - <~long-length>[/small]
+<~people>""" % _EMPTY
+
+
+class Preferences(qltk.UniqueWindow, EditDisplayPatternMixin):
```

Comment:
I'm not sure how to properly extend browsers.albums.Preferences without also invoking the AlbumList preferences.